pkgs/development/libraries/rocksdb/default.nix

```diff
@@ -68,7 +68,7 @@ stdenv.mkDerivation rec {
     echo "BUILD CONFIGURATION FOR SANITY CHECKING"
     cat make_config.mk
     mkdir -pv $static/lib/
-    mv -vi $out/lib/librocksdb.a $static/lib/
+    mv -vi $out/lib/${LIBNAME}.a $static/lib/
```

Comment:
The problem was that with `enableLite`, the library was called `librocksdb_lite.a` and not `librocksdb.a`, so the `mv` failed.